### ISSUE TITLE
whitespace sensitive parsing inside a query

### ISSUE THREAD

**Issue by hsheth2** [joshtemple/lkml#73]
Thanks for the amazing project @joshtemple :)

I've run into another issue, similar to https://github.com/joshtemple/lkml/issues/59. 

The following LookML works fine.

```lookml
explore: dataset_owners {  
  query: my_owner_query {
    sorts: [ownership_count: desc, urn: asc ]
  }
}
```

But adding a space after `desc` causes issues. This validates just fine in the Looker online IDE.

```lookml
explore: dataset_owners {  
  query: my_owner_query {
    sorts: [ownership_count: desc , urn: asc ]
  }
}
```

When parsing this with lkml==1.2.0, it fails with this error
```bash
$ lkml -v model_failure_1.lkml
...
SyntaxError: Unable to find a matching expression for '<whitespace>' on line 3
```

I've observed this for both `sorts` and `filters` within a query. Additionally, it also happens when the comma is on the following line.

**Comment by hsheth2** [comment]
Looks like this was fixed by #72 a while ago, and is working with 1.3.0b5.

I guess my question is when might we see another (non-prerelease) release?

**Comment by joshtemple** [comment]
Hey @hsheth2, thanks for the nudge! I'd forgotten to get a proper release out. I've just released `1.3.0`, which will have this fix in it.

**Comment by hsheth2** [comment]
Thanks @joshtemple!

Also, looking forward to your talk at dbt Coalesce on Monday :)
